src/components/results/loadflow/load-flow-result.tsx

```diff
@@ -192,23 +234,35 @@ export const LoadFlowResult: FunctionComponent<LoadflowResultProps> = ({
             overloadedEquipment.limitType === LimitTypes.CURRENT
     );
 
+    useEffect(() => {
+        if (currentViolations && isFetchComplete) {
+            setIsCurrentViolationReady(true);
+        }
+    }, [currentViolations, isFetchComplete]);
+
     const renderLoadFlowCurrentViolations = () => {
         const message = getNoRowsMessage(
             messages,
             currentViolations,
-            loadFlowStatus
+            loadFlowStatus,
+            isCurrentViolationReady
         );
         const rowsToShow = getRows(currentViolations, loadFlowStatus);
         return (
-            <CustomAGGrid
-                rowData={rowsToShow}
-                defaultColDef={defaultColDef}
-                enableCellTextSelection={true}
-                columnDefs={loadFlowCurrentViolationsColumns}
-                onGridReady={onGridReady}
-                getRowStyle={getRowStyle}
-                overlayNoRowsTemplate={message}
-            />
+            <>
+                <Box sx={{ height: '4px' }}>
```

Comment:
I think you should include  the box in the condition because after the load, a little space is created between the tab and the table: {  openLoaderCurrentTab &&  <Box sx={{ height: '4px' }}> ...

Reply:
the space is always here on purpose, this way when the loader appear the table doesn't "jump" down. It was requested this way at least.

Reply:
ok so nothing to do